config/packages/prod/monolog.yaml

```diff
@@ -8,7 +8,7 @@ monolog:
             buffer_size: 50 # How many messages should be saved? Prevent memory leaks
         nested:
             type: stream
-            path: "%kernel.logs_dir%/%kernel.environment%.log"
+            path: php://stderr
```

Comment:
Don't we need to wrap the value with quotes? Probably double colon on the same line may cause problems

Reply:
No need, this is valid YAML.